docs/reference/method/MongoDBGridFSBucket-delete.txt

```diff
@@ -21,9 +21,11 @@ Definition
 
       function delete($id): void
 
-   This method has the following parameters:
+Parameters
+----------
 
-   .. include:: /includes/apiargs/MongoDBGridFSBucket-method-delete-param.rst
```

Comment:
Where is located this file? I don't see it in the deletions

Reply:
The extracts and apiargs YAML files get split into RST files as part of the build process.